packages/core/src/external/aws/s3.ts

```diff
@@ -1,15 +1,42 @@
+import { PutObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import { getSignedUrl as getPresignedUrl } from "@aws-sdk/s3-request-presigner";
 import * as AWS from "aws-sdk";
 import dayjs from "dayjs";
 import duration from "dayjs/plugin/duration";
 import * as stream from "stream";
 
 dayjs.extend(duration);
 const DEFAULT_SIGNED_URL_DURATION = dayjs.duration({ minutes: 3 }).asSeconds();
+const UPLOAD_FILE_SIZE_LIMIT = 25_000_000; // 25MB
```

Comment:
not sure if we discussed this before, but I've seen files w/ ~20MB, so maybe adding a bit more room here would be a good idea? 50MB is not a big deal for us, I think we're not processing those files, right?

Reply:
i don't think we've discussed it, but it makes sense! especially `tiff` files tend to be pretty large